ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java

```diff
@@ -52,7 +57,11 @@ public UserDto userUpdate(UserDto userDto) throws IOException, InterruptedExcept
                 .PUT(HttpRequest.BodyPublishers.ofString(JsonMapper.toJson(userDto)))
                 .build();
         HttpResponse<String> response = null;
-        response = client.send(request, HttpResponse.BodyHandlers.ofString());
+        try {
+            response = client.send(request, HttpResponse.BodyHandlers.ofString());
+        } catch (IOException e) {
+            throw new IOException("IOException to remote address " + userUpdateAddress);
+        }
```

Comment:
I didn't get this solution, could you expand this?

Reply:
I want to add a message to the exception
I guess a better way to do this is:
```
} catch (IOException e) {
    ex.Data.Add("UserMessage", "IOException to remote address " + userUpdateAddress);
    throw;
}
```

Reply:
Changed to 
`} catch (IOException e) {
            throw new HttpClientIOException("IOException to remote address " + roomRequestAddress);
        }`